src/solver/yaml_helper.jl

```diff
@@ -1,24 +1,149 @@
 import YAML
+import ClimaAtmos as CA
 
 """
-    commands_from_yaml(yaml_file; filter_name=nothing)
-
-A vector of strings containing the `command`s in a (buildkite)
-yaml file, `yaml_file`. `filter_name` can be used to filter out
-required substrings from each elements of the vector.
-"""
-function commands_from_yaml(yaml_file; filter_name = nothing)
-    data = YAML.load_file(yaml_file)
-    steps = filter(x -> x isa Dict && haskey(x, "steps"), data["steps"])
-    cmds = map(x -> x["steps"], steps)
-    cmds = vcat(cmds...)
-    cmds = map(x -> x["command"], cmds)
-    cmds = vcat(cmds...)
-    cmds = map(x -> split(x, "\n"), cmds) # handle commands that run multiple scripts
-    cmds = vcat(cmds...)
-    cmds = map(x -> x isa AbstractString ? x : join(x), cmds)
-    if filter_name ≠ nothing
-        filter!(x -> occursin(filter_name, x), cmds)
+    default_config_dict()
+
+Loads the default configuration from files into a Dict for use in AtmosConfig().
+"""
+function default_config_dict()
+    default_config_file =
+        joinpath(pkgdir(CA), ".buildkite", "default_config.yml")
```

Comment:
```suggestion
        joinpath("..", "..", ".buildkite", "default_config.yml")
```
I think that's the right way to do the relative path? Or, we could try to figure out the `pkgdir(CA)` in a safe way (please see https://github.com/CliMA/policies/wiki/Software-Design-Patterns-(SDPs)#avoid-using-somepkg--import-somepkg-in-src).